renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
Oh wow, I didn't realise that GLES doesn't allow creating FBOs with sRGB textures. I understand the problem now, I assumed that was easily possible.

That seems like a massive missing feature - does that mean that anyone wanting to do gamma correct rendering on GLES has to do manual shader sRGB conversion to any offscreen targets?

It might be worth still leaving a comment explaining this as why we are unable to properly replay a capture using EGL_KHR_gl_colorspace (since although the user is able to create an sRGB backbuffer, we'll be forced to create a non-corrected FBO as you point out). I see what you mean though, it is tricky.